### ISSUE TITLE
Menu for uninstall apps

### ISSUE THREAD

**Issue by boistordu** [ChrisTitusTech/win10script#71]
It  would be nice to be able to select the apps we would like to uninstall like in the video. But when you are clicking essential tweaks you seem to uninstall every apps?
Unless you are telling me that every apps need to be uninstalled because they need windows store to work ?

**Comment by vasudeveloper001** [comment]
This. I simply tried with Cortana for now for safety to check what happens when the button is clicked. I would like to know what is safe and what is not.

The last time I did this, Sophia Script one I believe, it destroyed the ability to detect headphones on my laptop. This forces me to live with default Windows 10 with 3-4 GB RAM Usage on default.

**Comment by ChrisTitusTech** [comment]
Agreed, I am removing the bulk remove until I can get an easy select and remove built in.
